scripts/oneoff/generate-g-cloud-services.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+"""
+Script to generate, submit, approve and index fake services from the validation JSON schemas
+Only tested to work against g-cloud-9
+
+Pre-requisites:
```

Comment:
Thought about that and wondered if you'd bring it up. In some respects it seems nicer, and it does prevent having duplicated schemas... So I guess let's do it.